.semaphore/semaphore.yml

```diff
@@ -8,6 +8,14 @@ auto_cancel:
   running:
     when: "branch != 'master'"
 blocks:
+  - name: "Shell Check"
+    task:
+      jobs:
+      - name: Execute shellcheck
+        commands:
+          - checkout
+          - shellcheck ./.ci/*.sh && shellcheck release.sh
```

Comment:
This update put others execution in sequence after this.
Look at graph of semaphore - weird.
Let's keep spellcheck in Travis it looks ok in execution.

Reply:
I made a new github workflow for shellcheck. We have an open issue and ongoing work for shellcheck suppressions, I do not want to block PRs and worry about re-running CI (or local execution) to verify shellcheck execution succeeds.